raybot/commands/lunchshame.rb

```diff
@@ -32,11 +34,14 @@ class LunchShame < SlackRubyBot::Commands::Base
             end
           end
         end
-
-        client.say(channel: "C1TUV5XFA", text: picker + " is picking up today.")
-        if not meal_today
+        
+        unless meal_today
           client.say(channel: "C1TUV5XFA", text: "There is no lunch for today!")
-        elsif waiting_on.empty?
+          return
```

Comment:
Not sure if this will do something weird since we aren't in a function... You know better than I. 

In lorettalunch we just break when we want to exit and that seems to work?

Reply:
Eh, it seems to work fine.